Tasks/GitHubPRComment/githubComment/Utilities.ts

```diff
@@ -0,0 +1,4 @@
+export class AzureDevOpsVariables {
```

Comment:
Is this files being used? I couldn't find a reference to it.
If it is being used, I would recommend changing it like this:
```javascript
 public static readonly pullRequestNumber: string = getVariable('System.PullRequest.PullRequestNumber');
```
Or simply creating a class to handle all the task's inputs and variables